lib/libexfat.c

```diff
@@ -604,3 +621,10 @@ int exfat_set_volume_serial(struct exfat_blk_dev *bd,
 
 	return ret;
 }
+
+unsigned int exfat_calc_cluster_offset(struct exfat_blk_dev *bd,
+		unsigned int clu_off, unsigned int clu)
```

Comment:
I think the names of this function and  arguments are not clear. Shouldn't we reflect the device byte offset in the function name and sector unit in the clu_off argument?

Reply:
the function name and clu_off argument will rename to exfat_clus_to_blk_dev_off() and clu_off_sectors.